packages/react/src/evm/utils/headers.ts

```diff
@@ -0,0 +1,30 @@
+import packageJson from "../../../package.json";
+import { detectOS } from "./isMobile";
+
+export function getAnalyticsGlobals() {
```

Comment:
who's calling this in react? shouldn't that still be called in the ThirdwebProvider?